src/Workspaces/VisualBasic/Portable/LanguageServices/VisualBasicSemanticFactsService.vb

```diff
@@ -256,6 +257,18 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Return Nothing
         End Function
 
+        Public Function GetAwaitExpressionMethod(model As SemanticModel, node As SyntaxNode) As IMethodSymbol Implements ISemanticFactsService.GetAwaitExpressionMethod
+            If node.IsKind(SyntaxKind.AwaitExpression) Then
+                Dim awaitExpression = DirectCast(node, AwaitExpressionSyntax)
+                Dim builder = ArrayBuilder(Of IMethodSymbol).GetInstance()
+                Dim info = model.GetAwaitExpressionInfo(awaitExpression)
+
+                Return info.GetAwaiterMethod
+            End If
+
+            Return Nothing
+        End Function
```

Comment:
ok.  it feels lke we don't need this method.  We already have IsAwaitExpression.  So all we need to do is call model.GetAwaitExpressionInfo() if we ahve an await expression.  The rest of the code can all be shared by the caller.

Reply:
I left this method as-is, due to the problem with `AwaitExpressionInfo`.